src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -368,10 +368,18 @@ std::string statisticsTreat
     out += TAG_ADD_COUNTER("logRequests", noOfLogRequests);
   }
 
+#if 0
+  //
+  // The valgrind test suite uses REST GET /version to check that the broker is alive
+  // This fact makes the statistics change and some working functests fail under valgrindTestSuite
+  // due to the 'extra' version-request in the statistics.
+  // Removing version-requests from the statistics 'report' fixes this problem.
+  //
   if (noOfVersionRequests != -1)
   {
     out += TAG_ADD_COUNTER("versionRequests", noOfVersionRequests);
   }
+#endif
```

Comment:
I left this "#if 0 / #endif" for a discussion on how to solve this.

Instead of doing this, I have solved the problem by adding a few "GET  /version  in XML" in functests and REGEX((1|2)) or similar in REGEXPECT part.

The 'problem' is due to the function stat starts the broker (actually, the function that awaits the broker start), sends a GET /version "in XML" to make sure the broker is running. And, of course, this alters the request statistics ...
One count in func-test, another count under valgrind.